src/theory/strings/theory_strings_rewriter.cpp

```diff
@@ -653,13 +653,13 @@ Node TheoryStringsRewriter::rewriteLoopRegExp(TNode node)
   }
   TNode n1 = node[1];
   NodeManager* nm = NodeManager::currentNM();
-  CVC4::Rational RMAXINT(LONG_MAX);
+  CVC4::Rational rMaxInt(UINT32_MAX);
```

Comment:
Maybe we should define the maximum value in one place and then refer to it from everywhere?